.github/scripts/questbook-i18n.py

```diff
@@ -1,60 +1,119 @@
 import json
+import logging
+from abc import ABCMeta, abstractmethod
+from pathlib import Path
 from typing import TextIO
-from abc import ABCMeta
 
-DEFAULT_QUESTS_PATH = 'config/betterquesting/DefaultQuests.json'
-EN_US_LANG_PATH = 'config/txloader/load/minecraft/lang/en_US.lang'
-DEFAULT_QUESTS_US_PATH = 'config/betterquesting/DefaultQuests-us.json'
+DEFAULT_QUESTS_BASE_DIR_PATH = Path('config/betterquesting/DefaultQuests')
+QUEST_LINES_ORDER_PATH = DEFAULT_QUESTS_BASE_DIR_PATH / 'QuestLinesOrder.txt'
+QUEST_LINES_DIR_PATH = DEFAULT_QUESTS_BASE_DIR_PATH / 'QuestLines'
+QUESTS_DIR_PATH = DEFAULT_QUESTS_BASE_DIR_PATH / 'Quests'
+EN_US_LANG_PATH = DEFAULT_QUESTS_BASE_DIR_PATH / 'en_US.lang'
```

Comment:
If the intention is to fully replace the `en_US.lang` generated by BQ, then I would suggest putting `en_US.lang` somewhere that is not in the `DefaultQuests/` directory. Possible ideas:

1. `config/betterquesting`
2. `config/txloader/load/betterquesting/lang`

The reason for this is the `DefaultQuests/` directory gets deleted every time `/bq_admin default save` is run, and if this file is no longer being generated by BQ, that will cause PRs to needlessly delete the file repeatedly only to have it get re-created by the workflow.

Reply:
Changing the directory where en_US.lang is located would require modifying the BQ code, so it's up to you to choose a path.

Reply:
Once BQ code is modified to no longer generate `en_US.lang`, wouldn't it be determined by line 11 here?

Reply:
> 5. Just drop the translated lang file into config/txloader/load/betterquesting/lang to apply the translations. In previous versions of BetterQuesting, it was necessary to load a custom version of the quest book data for translations to work, but this is no longer needed.

Oh, I forgot that all the translated files are in the `config/txloader/load/betterquesting/lang`. Alright, then let's put `en_US.lang` in there as well.

Reply:
done